modules/webgl/src/index.js

```diff
@@ -8,19 +8,22 @@
 export {lumaStats} from './init';
 
 // TODO - should we reexport these?
-export {resetParameters, getParameters, setParameters, withParameters} from '@luma.gl/gltools';
-
-export {createGLContext, resizeGLContext, instrumentGLContext} from './context/context';
+export {
+  resetParameters,
+  getParameters,
+  setParameters,
+  withParameters,
+  isWebGL,
+  isWebGL2
+} from '@luma.gl/gltools';
```

Comment:
I think the atom modules should avoid re-exporting APIs from their dependencies. Whoever importing these from the webgl module should switch to importing from gltools, so that we are perfectly clear where things live.

Reply:
Sounds good to me.